creator-node/src/services/stateMachineManager/stateMonitoring/fetchCNodeEndpointToSpIdMap.jobProcessor.js

```diff
@@ -11,8 +11,9 @@ const NodeToSpIdManager = require('../CNodeToSpIdMapManager')
 module.exports = async function ({ logger }) {
   let errorMsg = ''
   try {
+    const audiusLibs = await initAudiusLibs()
```

Comment:
im' supporting of trying this change, the only thing i'm a bit concerned about is the extra load every job initializing libs will place on this. for example this will have to do a full discovery selection and make health check reqs to all nodes etc. could we pair this with a watered down version of libs to prune stuff we don't need?

Reply:
looks like the things we need are:
- `contracts.UserReplicaSetManagerClient.updateReplicaSet`
- `ServiceProvider.autoSelectCreatorNodes`
- `ethContracts.getServiceProviderList`

I traced through the various top-level config options, and `discoveryProviderConfig` was the only one we can get away with not passing for the above functions. that will avoid the heavy discovery selection at least